build/teamcity-sqllogictest.sh

```diff
@@ -16,13 +16,13 @@ export BUILDER_HIDE_GOPATH_SRC=0
 build/builder.sh \
   stdbuf -oL -eL \
   make test TESTFLAGS="-v -bigtest" TESTTIMEOUT='24h' PKG='./pkg/sql/logictest' TESTS='^TestSqlLiteLogic$$' 2>&1 \
-  | tee "artifacts/${config}.log" \
+  | tee "artifacts/logictest.log" \
   | go-test-teamcity
 
 # Need to specify the flex-types flag in order to skip past variations that have
 # numeric typing differences.
 build/builder.sh \
   stdbuf -oL -eL \
   make test TESTFLAGS="-v -bigtest -config local-opt,fakedist-opt -flex-types" TESTTIMEOUT='24h' PKG='./pkg/sql/logictest' TESTS='^TestSqlLiteCorrelatedLogic$$' 2>&1 \
-  | tee "artifacts/${config}.log" \
+  | tee "artifacts/logictest.log" \
```

Comment:
Does this overwrite the file above? Seems like it needs to go to a different filename.

Reply:
Derp yes. Fixed.